rules/netacea_mitigate.tcl

```diff
@@ -0,0 +1,121 @@
+when HTTP_REQUEST {
```

Comment:
It's pretty hard to validate these by inspection alone haha, is it possible to unit tests them?

Reply:
So, after some research. Unit tests support for TCL is pretty slim and not very implemented yet. So I'm going to say no it's not possible currently.